### PR TITLE
- Starting pull request

### PR DESCRIPTION
**Pull request by agonosmani**

### CODE REVIEW

**Comment by nortexmkd** [comment]
good!

**Comment by Kiko117001** [comment]
i like dis very gud yes

File: newtextfile.html [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+    <h1>This is a new file</h1>
+    <h2>Please accept this into the main branch</h2>
```
Comment (Kiko117001) [inline newtextfile.html:9]: i will not accept this

**Review by Kiko117001 — APPROVED** [review]

**Review by jovanasmilevska — APPROVED** [review]

File: newtextfile.html [hunk 2]
```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
```
Comment (nortexmkd) [inline newtextfile.html:5]: change title

**Review by nortexmkd — APPROVED** [review]

**Pull request closed without merging**